language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSPackageLoader.java

```diff
@@ -163,15 +168,22 @@ private List<PackageInfo> getPackagesFromRepository(PackageRepository repository
                 PackageOrg packageOrg = PackageOrg.from(key);
                 PackageName packageName = PackageName.from(nameComponent);
                 PackageVersion pkgVersion = PackageVersion.from(version);
-                PackageDescriptor pkdDesc = PackageDescriptor.from(packageOrg, packageName, pkgVersion);
-                ResolutionRequest request = ResolutionRequest.from(pkdDesc, PackageDependencyScope.DEFAULT);
 
-                Optional<Package> repoPackage = repository.getPackage(request,
-                        ResolutionOptions.builder().setOffline(true).build());
-                repoPackage.ifPresent(pkg -> packages.add(new PackageInfo(pkg)));
+                try {
+                    PackageDescriptor pkdDesc = PackageDescriptor.from(packageOrg, packageName, pkgVersion);
+                    ResolutionRequest request = ResolutionRequest.from(pkdDesc, PackageDependencyScope.DEFAULT);
+
+                    Optional<Package> repoPackage = repository.getPackage(request,
+                            ResolutionOptions.builder().setOffline(true).build());
+                    repoPackage.ifPresent(pkg -> packages.add(new PackageInfo(pkg)));
+                } catch (Throwable e) {
+                    clientLogger.logTrace("Failed to resolve package "
```

Comment:
It's better if we logged an error. But ok for now.

Reply:
@malinthar let's push this as a separate PR.